src/c++/perf_analyzer/data_loader.cc

```diff
@@ -343,7 +344,14 @@ DataLoader::GetInputData(
         string_data = &it->second;
         *batch1_size = string_data->size();
       }
-      *data_ptr = (const uint8_t*)&((it->second)[0]);
+
+      // FIXME this is the fix. It is ugly
+      if (it->second.size()) {
```

Comment:
This is the fix, although it is ugly. 
it->second is the data in the form of a vector<char>. In the case that the vector is empty, then `[0]` used to be nullptr. Now, the code will return non-null pointer.

Reply:
@matthewkotila I created a follow-up story to add the return value and converting to a struct. The change was too big to be included in this story.